internal/tree.go

```diff
@@ -294,3 +300,58 @@ func updatePsTreeCommToCmdline(checkpointOutputDir string, psTree *crit.PsTree)
 	}
 	return nil
 }
+
+// Taken from the CRI API
+type mountAnnotations struct {
+	ContainerPath     string `json:"container_path,omitempty"`
+	HostPath          string `json:"host_path,omitempty"`
+	Readonly          bool   `json:"readonly,omitempty"`
+	SelinuxRelabel    bool   `json:"selinux_relabel,omitempty"`
+	Propagation       int    `json:"propagation,omitempty"`
+	UidMappings       []*int `json:"uidMappings,omitempty"`
+	GidMappings       []*int `json:"gidMappings,omitempty"`
+	RecursiveReadOnly bool   `json:"recursive_read_only,omitempty"`
+}
+
+func addPodInfoToTree(tree treeprint.Tree, info *checkpointInfo) {
+	podTree := tree.AddBranch("Metadata")
+	podTree.AddBranch(fmt.Sprintf("Name: %s", info.containerInfo.Pod))
+	podTree.AddBranch(fmt.Sprintf("Namespace: %s", info.containerInfo.Namespace))
```

Comment:
It might be better to skip these annotations if the corresponding value is empty:
```
Displaying container checkpoint tree view from /tmp/chkpt.tar.gz

looper
├── Image: docker.io/library/busybox:latest
├── ID: 50214caf805e3011f8db0fb2a671ab911cc12d7a8d3fc9dc7b769f71f9777a8c
├── Runtime: crun
├── Created: 2024-10-17T16:51:27+01:00
├── Engine: Podman
├── Checkpoint size: 319.3 KiB
│   └── Memory pages size: 288.0 KiB
└── Metadata
    ├── Name: 
    ├── Namespace: 
    └── Annotations
        ├── io.container.manager: libpod
        ├── org.opencontainers.image.stopSignal: 15
        ├── org.systemd.property.KillSignal: 15
        └── org.systemd.property.TimeoutStopUSec: uint64 10000000
```

Reply:
done